### ISSUE TITLE
RJ bounce improvements

### ISSUE THREAD

**Issue by hexaflexahexagon** [momentum-mod/game#1154]
**Is your feature request related to a problem? Please describe.**
RJ (and SJ) bounces / bouncehops are height dependent, require precise setups that need a plugin/spreadsheet to find, and need janky height measurements to create as a mapper. Fixing this involves two steps:
1. Set `sv_considered_on_ground 1` by default. Was previously set to 2 to keep legacy bounces but ideally all old bounces will be replaced by `trigger_momentum_onehop` setups. 
2. Add some player flag / filter that can enabled upon hitting a trigger (such as a floor or ceiling) and disabled upon being hit by a rocket. This is necessary to prevent the player from shooting themselves downwards into a onehop trigger, something that is not desired and is not possible in TF2. 

While 2 is most likely technically possible currently, considering this is a staple mechanic of many jump maps it should be simplified as much as is reasonable.

**Comment by braem** [comment]
Does `trigger_momentum_onehop` work with crouch bounces?

**Comment by hexaflexahexagon** [comment]
> Does `trigger_momentum_onehop` work with crouch bounces?

Yes it does, all the entity does is teleport the player after being within the trigger for x seconds (e.g. 0.015), it works the same for crouch, uncrouch, and bouncehops.

However something I wasn't aware of is that ILDPRUT has their own implementation idea for this bounce issue which would most likely be cleaner than this one, so this issue should either be reserved for just them or whoever decides to tackle it should talk to them before doing anything else.

https://discord.com/channels/235111289435717633/602643431453360148/777809749793964042

**Comment by Gocnak** [comment]
@chovelyoukai Did you tackle this work recently?
